### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Istvan91**
Removing lots of trailing whitespace + applying some markdownlint suggestions. So just a bunch of formatting stuff without any semantic changes.

### CODE REVIEW

File: docs/multicast.md [hunk 1]
```diff
@@ -165,7 +170,7 @@ sending leave and the last multicast packet received for this group. Multicast p
 received for the leaved group after first packet of joined group is received are counted
 as overlap.
 
-The following <<Configuration>> output shows an example for the `igmp` section
```
Comment (GIC-de) [inline docs/multicast.md:168]: Please do not remove the cross references (`<<..>>`).

File: docs/multicast.md [hunk 2]
```diff
@@ -80,25 +81,27 @@ example below:
...
+Using RAW streams allows to generate streams distributed over multiple network interfaces.
 Setting the `destination-ipv4-address` to an multicast IPv4 address is enough to generate
 proper multicast streams. All headers including the BNG Blaster header will be automatically
-set for multicast. Therefore such streams can be also used to measure the IGMP join and leave 
+set for multicast. Therefore such streams can be also used to measure the IGMP join and leave
 delay.
 
 ## Manual Join/Leave Testing
 
-It is possible to join and leave multicast groups manually using the <<Control Socket>> as
```
Comment (GIC-de) [inline docs/multicast.md:91]: Please do not remove the cross references (`<<..>>`).

File: docs/multicast.md (hunk 1 above)
Comment (Istvan91) [reply docs/multicast.md:168]: have you lactually ooked at the result in github? it is simply rendered as <> so whatever that intended - it does not work.

File: docs/multicast.md (hunk 2 above)
Comment (Istvan91) [reply docs/multicast.md:91]: same as above

**Pull request merged**